config/ipam/kustomization.yaml

```diff
@@ -3,7 +3,7 @@ kind: Kustomization
 
 # When updating the release, update also the image tag in image_patch.yaml
 resources:
-- https://github.com/metal3-io/ip-address-manager/releases/download/v1.6.0/ipam-components.yaml
+- metal3-ipam-components.yaml
```

Comment:
this is a problem, with original version we have a way to deploy ipam with specific version, usually it is aligned with CAPM3, but now we are hardcoding it. What are you trying to achieve with that approach?

Reply:
I think those files in `/config` were changed when I've run the test and I have should not commit those. Reverted.
Sorry for that.